### ISSUE TITLE
late binding view introspection conflicts with drop statements on Redshift

### ISSUE THREAD

**Issue by drewbanin** [fishtown-analytics/dbt#1305]
### Issue description
A drop statement run concurrently with a query to `pg_get_late_binding_view_cols()` will cause the `pg_get_late_binding_view_cols()` query to fail with something like:
```
ERROR:  table 6616765 dropped by concurrent transaction
```

I posted this (apparent) Redshift bug to the AWS Redshift forum, but also wanted to track it here. Check out the post for more info: https://forums.aws.amazon.com/thread.jspa?threadID=298770

This can be reproduced using the scripts here: https://gist.github.com/drewbanin/98beec96edadfe25398a79a8274b2cdb

I think we're blocked on Redshift fixing this, but very open to any clever ideas here. One way to mitigate this would be to cache introspective queries during parsing, which I will create a separate issue for.

**Comment by drewbanin** [comment]
The Redshift forum link above indicates that this bug is fixed! Closing (and crossing my fingers 🤞)
